stdlib/public/Distributed/DistributedActor.swift

```diff
@@ -46,20 +46,23 @@ public protocol DistributedActor: AnyActor, Identifiable, Hashable, Codable {
     ///   associated with.
     init(transport: ActorTransport)
 
-    /// Resolves the passed in `address` against the `transport`,
-    /// returning either a local or remote actor reference.
+    @available(*, deprecated, renamed: "SomeDistributedActor.resolve(_:using:)")
+    init(resolve id: AnyActorIdentity, using transport: ActorTransport) throws
```

Comment:
let's completely ignore the existence of this; I'm only keeping it around so we don't have to remove useful tests that we have already.

Once the resolve function exists I'll remove all backing infra for this init.